CHANGELOG.md

```diff
@@ -21,14 +66,13 @@ These should not affect the functionality of the module.
 
 **Fixed bugs:**
 
-- Support for Rundeck 3.1 [\#419](https://github.com/voxpupuli/puppet-rundeck/issues/419)
+- Rundeck 3.1 does not require rundeck-config package anymore [\#419](https://github.com/voxpupuli/puppet-rundeck/issues/419)
 - groupdel: cannot remove the primary group of user 'rundeck' [\#199](https://github.com/voxpupuli/puppet-rundeck/issues/199)
 
 **Merged pull requests:**
 
-- Token duration must be a string [\#424](https://github.com/voxpupuli/puppet-rundeck/pull/424) ([danifr](https://github.com/danifr))
+-  Fix tests for \#413  [\#424](https://github.com/voxpupuli/puppet-rundeck/pull/424) ([danifr](https://github.com/danifr))
```

Comment:
Previous label was more insightful for end-users.

Reply:
the generator had multithreading issues in the past. the new title is the one also set for the PR. I think it's fine now.